Source/CuckooGeneratorFramework/Generator.swift

```diff
@@ -70,12 +70,12 @@ public struct Generator {
         output += ""
         output += "\(getAccessibilitySourceName(token.accessibility))\(token.overriding ? "override " : "")var \(token.name): \(token.type) {"
         output += "    get {"
-        output += "        return manager.getter(\"\(token.name)\", original: observed.map { o in return { () -> \(token.type) in o.\(token.name) } })()"
+        output += "        return manager.getter(\"\(token.name)\", original: observed.map { o in return { () -> \(token.type) in o.\(token.name) } })"
         output += "    }"
         
         if token.readOnly == false {
             output += "    set {"
-            output += "        manager.setter(\"\(token.name)\", value: newValue, original: { self.observed?.\(token.name) = $0 })(newValue)"
```

Comment:
Why did you remove the call at the end? I know it might not be useful now that we removed support for `@noescape` closures, but once Swift 3 is out, we might need to roll it back in.